Source/CronosPlayUnreal/CronosPlayUnreal.Build.cs

```diff
@@ -14,6 +14,9 @@ public class CronosPlayUnreal : ModuleRules {
   }
 
   public CronosPlayUnreal(ReadOnlyTargetRules Target) : base(Target) {
+    bEnableExceptions = true;
```

Comment:
what's this setting for?

Reply:
To enable try catch feature (which is discouraged by epic game). Without this, build will fail.

Reply:
To remove this, we have to remove all try catch statements in our source codes.